### DISCUSSION TITLE
Demo of MethodsHub - Minimal Example of Tutorial in Markdown with R and System Requirements

### DISCUSSION THREAD

**Discussion by giscus** [GESIS-Methods-Hub/methodshub#32] [General]
# Demo of MethodsHub - Minimal Example of Tutorial in Markdown with R and System Requirements

http://svma-css-methodshub01.gesis.intra/demo/GESIS-Methods-Hub/minimal-example-qmd-rstats-units/

**Comment by rgaiacs** [comment]
Thanks for the clarification about dependencies.
